pymod/chemistry/atom_complex_calc.py

```diff
@@ -0,0 +1,122 @@
+import math
+import sys # noqa
+sys.path.insert(0, '..') # noqa
+
+from geometry import (
+        Coordinate,
+        Surface,
+        Constants,
+    )
+
+from . import (
+        Atom,
+        AtomList,
+        CellList,
+    )
+
+class AtomComplexCalc:
+    '''
+    Port of Vacc
+    '''
+
+    def __init__(self, alist: AtomList, clist: CellList, surface_density: float):
+        '''
+        Note: This is not just a port of the ctor for Vacc, but also replaces
+        Vacc_storeParms. Vacc_storeParms seems to be simply an extension to the
+        constructor, so it's job is replicated here.
+        
+        If it is found that Vacc_storeParms is not just an extension for the 
+        constructor, it should be moved to another method.
+        '''
+        self.clist = clist
+        self.alist = alist
+        self.surface_density = surface_density
+        max_radius = alist.max_radius + clist.max_radius
+        max_area = 4. * ( max_radius ** 2 ) * math.pi
+        nsphere = math.ceil(max_area * surface_density)
+
+        # TODO: calculate reference shpere (see VaccSurf_refSphere)
+
+    @property
+    def stride(self) -> Coordinate:
+        return self.clist.stride
+
+    def accessable_outside_inflated_venderwalls_rad(self, center: Coordinate,
```

Comment:
This function should be renamed to `accessible_outside_inflated_vanderwaals`.  See https://en.wikipedia.org/wiki/Johannes_Diderik_van_der_Waals